src/System.Drawing.Common/tests/Drawing2D/GraphicsPathTest.cs

```diff
@@ -0,0 +1,3294 @@
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Clone_Success()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPath clone = Assert.IsType<GraphicsPath>(gp.Clone()))
+            {
+                Assert.Equal(FillMode.Alternate, clone.FillMode);
+                AssertEmptyGrahicsPath(clone);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Reset_Success()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.Reset();
+
+                Assert.Equal(FillMode.Alternate, gp.FillMode);
+                AssertEmptyGrahicsPath(gp);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GraphicsPath_FillModeChange()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.FillMode = FillMode.Winding;
+                Assert.Equal(FillMode.Winding, gp.FillMode);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GraphicsPath_InvalidFillMode_ThrowsInvalidEnumArgumentException()
+        {
+            Assert.Throws<InvalidEnumArgumentException>(() => new GraphicsPath().FillMode = ((FillMode)int.MaxValue));
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void PathData_ReturnsExpected()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                Assert.Equal(0, gp.PathData.Points.Length);
+                Assert.Equal(0, gp.PathData.Types.Length);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void PathData_CannotChange()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.AddRectangle(new Rectangle(1, 1, 2, 2));
+                Assert.Equal(1f, gp.PathData.Points[0].X);
+                Assert.Equal(1f, gp.PathData.Points[0].Y);
+
+                gp.PathData.Points[0] = new Point(0, 0);
+                Assert.Equal(1f, gp.PathData.Points[0].X);
+                Assert.Equal(1f, gp.PathData.Points[0].Y);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void PathPoints_CannotChange()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                gp.AddRectangle(new Rectangle(1, 1, 2, 2));
+                Assert.Equal(1f, gp.PathPoints[0].X);
```

Comment:
Maybe `Assert.Equal(new PointF(1, 1), gp.PathPoints[0])` would be clearer here and elsewhere